hkm/management/commands/clean_unused_data.py

```diff
@@ -0,0 +1,46 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from hkm.models.models import TmpImage, Feedback, ProductOrder
+from datetime import timedelta
+from django.utils import timezone
+
+DEFAULT_DAYS = 365
+
+class Command(BaseCommand):
+    help = "Clean unused data from application."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-d", "--days", type=int, help="Specifies the time in days for which old data is deleted", default=DEFAULT_DAYS
+        )
+
+    def handle(self, *args, **kwargs):
+        days = kwargs['days']
+        if not days:
+            days = DEFAULT_DAYS
+
+        self.stdout.write("Old data cleaning started! Removing data older than " + str(days) + " days.")
+        counted_date = timezone.now() - timedelta(days=days)
+
+        # Find and delete unused users
+        unused_users = User.objects.filter(last_login__lte=counted_date)
+        unused_users_count = unused_users.count()
```

Comment:
The `count()` is unnecessary in all cases, since `delete()` should return the number of deleted objects. Also, doing it this way is not an atomic operation + creates an extra query to the database. See https://docs.djangoproject.com/en/1.11/topics/db/queries/#deleting-objects for reference.

You should be able to get away with just `result = User.objects.filter(...).delete()` or similar.